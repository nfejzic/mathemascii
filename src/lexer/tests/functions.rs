use super::Snapshot;

use super::super::*;

super::test_snap!(sin_cos, "sincos");
super::test_snap!(f_g, "fg");
super::test_snap!(ln_log, "ln logln");
super::test_snap!(too_long, "abcdefghijklmnopqrstuvwxyf");
super::test_snap!(multiple, "sincoshfg");
super::test_snap!(precedence, "gammag");
