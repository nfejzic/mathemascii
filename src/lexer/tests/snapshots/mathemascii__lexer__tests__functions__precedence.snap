---
source: src/lexer/tests/functions.rs
expression: "Snapshot((input, tokens))"
---
gammag

gamma
^^^^^ -> Greek(Gamma) at: 0 -> 5

     g
     ^ -> Function(G) at: 5 -> 6


